code/modules/food_and_drinks/drinks/drinks.dm

```diff
@@ -265,6 +265,13 @@
 	list_reagents = list(/datum/reagent/consumable/ethanol/beer = 30)
 	foodtype = GRAIN | ALCOHOL
 
+/obj/item/reagent_containers/food/drinks/syndicatebeer
+	name = "suspicous looking beer"
+	desc = "The bottle is painted black with an S on the label. Looks like some sort of enemy corporation."
```

Comment:
```suggestion
	desc = "The bottle is painted black with an S on the label. Looks like some sort of enemy corporation."
```

I can't entirely explain why, but I don't really like how the last bit is worded, "Looks like some sort of enemy corporation." It's common in-universe knowledge who the Syndicate Organization is, not sure if we need to explain it here as well.

This is an overall really minor gripe though

Reply:
Actually, I just realized, Syndicate players are going to be buying this right? Wouldn't they know who they work for? Also would they label their own organization as an "enemy corporation?"

Reply:
i didnt think about that, good point